templates/solaris-0.1.2/src/Algorithm.py

```diff
@@ -3,16 +3,85 @@
 # API calls will begin at the apply() method, with the request body passed as 'input'
 # For more details, see algorithmia.com/developers/algorithm-development/languages
 
+import numpy as np
+import solaris as sol
+from shapely.wkt import loads
+
+def rescale_auto():
```

Comment:
Maybe add some comments on what this does?

Reply:
I have no idea what it does, this was provided to me as canary functions by the customer; this package set will never be public facing, so I don't see that as a terrible thing